ext/kv/01_db.ts

```diff
@@ -312,6 +315,10 @@ class Kv {
     core.close(this.#rid);
     this.#closed = true;
   }
+
+  get rid() {
```

Comment:
Do we expose the `rid` in a public API elsewhere? cc @bartlomieju / @lucacasonato

Reply:
It is exposed for a few resources (e.g. `Conn`, `HttpConn`)

Reply:
It is exposed in a few resources, but the idea was that we're gonna hide in in Deno 2.

Reply:
I mainly added it here for a test. Is there a better way to do this?